app/dashboard/models.py

```diff
@@ -1294,7 +1294,7 @@ def get_slack_repos(self, join=False):
 
         """
         if join:
-            repos = ','.join(self.slack_repos)
+            repos = ', '.join(self.slack_repos)
```

Comment:
Change made for the presentation of repos - "slack_repo_1, slack_repo_2" looks better than "slack_repo_1,slack_repo_2".

Reply:
i just worry that if someone does a comma deliminted list and doesnt include the ` ` (space) then it wont correctly split/join

Reply:
We could just split on `,` and `trim` each element.